File: backend/SnowConeOperations/SnowConeOperations.Api/Models/InventoryItem.cs
namespace SnowConeOperations.Api.Models
{
    public class InventoryItem
    {
        public int Id { get; set; }
        public string ItemName { get; set; } = string.Empty;
        public int Quantity { get; set; }
        public string UnitMeasured { get; set; } = string.Empty;
        public DateTimeOffset CreatedAt { get; set; }

        public DateTimeOffset UpdatedAt { get; set; }
    }
}

File: frontend/src/types/InventoryItem.ts
export interface InventoryItem {
  id: number;
  itemName: string;
  quantity: number;
  unitMeasured: string;
  createdAt: string;
  updatedAt: string | null;
}

export interface InventoryItemUpdate{
    itemName: string;
    quantity: number;
    unitMeasured: string;
}

export interface InventoryItemCreate{
  itemName: string;
  quantity: number;
  unitMeasured: string;
}